modules/jsdom/src/polyfills/events/keyboard.ts

```diff
@@ -31,7 +31,13 @@ import {
 
 export function keyboardEvents(window: DOMWindow) {
   const keys        = keyUpdates(window);
-  const specialKeys = keys.pipe(filter(isSpecialKey));
+  const specialKeys = keys.pipe(filter(isSpecialKey)).pipe(flatMap(function*(keyEvt) {
```

Comment:
What is the `*` for here in `function*`?

Reply:
That indicates this a [generator function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator).